src/components/m-datepicker/js/datepicker.js

```diff
@@ -89,7 +89,12 @@ export default class Datepicker {
     this.unOkButtonListenerEnd = on(this.wcNode.querySelector('.js-datepicker__button__Ok'), EVENTS.CLICK, () => {
       const year = this.datepickerBody.getAttribute('year');
       const month = this.datepickerBody.getAttribute('month');
-      const day = this.datepickerBody.getAttribute('value');
+      const dayAsValue = this.datepickerBody.getAttribute('value'); // new value - after click on a day
+      let dayAsDay = this.datepickerBody.getAttribute('day'); // prev value - prior click on a day
```

Comment:
please use `js/get-attribute.js` helper to do this.

Reply:
hmh, why isn't `getAttribute` helper used?